b2sdk/raw_api.py

```diff
@@ -548,9 +548,9 @@ def test_raw_api_helper(raw_api):
     this test will break and we'll have to do something about
     it.
     """
-    account_id = os.environ.get('TEST_ACCOUNT_ID')
-    if account_id is None:
-        print('TEST_ACCOUNT_ID is not set.', file=sys.stderr)
+    application_key_id = os.environ.get('TEST_APPLICATION_KEY_ID')
```

Comment:
this requires changes in the build environment, please make sure you ask Brian Beach to adjust the CI configuration to provide the key on both env variables during the transition period

Reply:
I thought that the CI configuration only supplied to strings which it mapped to environment variable names. I will check with Brian ty.

Reply:
Yes. For the transition period we should export this as **both** `TEST_ACCOUNT_ID` and `TEST_APPLICATION_KEY_ID`.

Reply:
@bwbeach should I make a check for either `TEST_ACCOUNT_ID` or `TEST_APPLICATION_KEY_ID`?